### DISCUSSION TITLE
故地重游 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#455] [General]
# 故地重游 - Miao Yu | 于淼

https://yufree.cn/cn/2024/03/14/xian/

**Comment by XiangyunHuang** [comment]
之前看到一则视频，[【铁头惩恶扬善帮西安不夜城演员讨薪】](https://www.bilibili.com/video/BV1Pw4m1R7oD/) ，真是没想到不夜城背后会发生这种事。

**Reply by yufree** [reply]
确实没想到，西安不夜城演员很辛苦，在夜间外面零下的温度在台上演出，按人流量跟消费水平也不会缺钱，有可能公司搞了别的房产或城投业务赚的钱被拿去填窟窿了，应该要有简易法律流程来走，不然按闹分配早晚要出事。
